src/neptune_scale/__init__.py

```diff
@@ -442,7 +442,7 @@ def log_metrics(
         """
         self.log(step=step, timestamp=timestamp, metrics=data)
 
-    def log_configs(self, data: Optional[Dict[str, Union[float, bool, int, str, datetime, list, set]]] = None) -> None:
+    def log_configs(self, data: Optional[Dict[str, Union[float, bool, int, str, datetime]]] = None) -> None:
```

Comment:
does logging these type actually fail? 
if so, why were they added here in the first place

Reply:
I get a `TypeError: bad argument type for built-in operation` when trying to log dicts with lists/sets as values

```
run.log_configs({"key": [1, 2, 3]})
run.log_configs({"key": {1, 2, 3}})
```

The [API ref](https://github.com/neptune-ai/neptune-client-scale?tab=readme-ov-file#log_configs) also does not include list and set

Reply:
some mypy check failed but apart from that it looks like a good fix to me then